3.7/graphs-satellite-graphs.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: default
+description: Graphs synchronously replicated to all servers, available in the Enterprise Edition
+title: ArangoDB Satellite Graphs
+---
+Satellite Graphs
+=====================
+
+{% hint 'info' %}
+Satellite Graphs are only available in the
+[**Enterprise Edition**](https://www.arangodb.com/why-arangodb/arangodb-enterprise/){:target="_blank"},
+and the [**ArangoDB Cloud**](https://cloud.arangodb.com/){:target="_blank"}.
+{% endhint %}
+
+Satellite Graphs are the natural extension of the concept of Satellite
+collections to graphs. All of the usual benefits and caveats apply.
+
+When doing joins involving graph traversals, shortest paths, or k-shortest paths
+in an ArangoDB cluster, data has to be exchanged between different servers. In particular
+graph traversals are usually executed on a Coordinator, because they need global information. 
+
+This results in a lot of network traffic and slow query execution.
+
+Satellite graphs are synchronously replicated to all DB-Servers that are part of
+a cluster, which enables DB-Servers to execute parts of joins locally.
+
+This greatly improves performance for such joins.
+
+To create a satellite graph in arangosh, use the `satelliteGraph` module.
+
+    arangosh> satGM = require("@arangodb/satellite-graph")
+    arangosh> satGM._create("satelliteGraph", [ satGM._relation("edges", "vertices", "vertices") ], [], {});
+
+You can also create vertex and edge collections manually, and set their `replicationFactor` to
+`satellite`
+
+    arangosh> db._createDocumentCollection("vertices2", { replicationFactor: "satellite" });
+    arangosh> db._createEdgeCollection("edges2", { replicationFactor: "satellite" });
```

Comment:
That alone does not suffice; `distributeShardsLike` has to be set to the leading collection of the graph, which is the first vertex collection added to it.

Reply:
Also, a line how these would be added to the actual graph would be good.